moveit_experimental/moveit_jog_arm/include/moveit_jog_arm/jog_calcs.h

```diff
@@ -81,7 +81,7 @@ class JogCalcs
   bool getCommandFrameTransform(Eigen::Isometry3d& transform);
 
   /** \brief Pause or unpause processing jog commands while keeping the timers alive */
-  void setPaused(bool paused);
+  void setPaused(const bool paused);
```

Comment:
I've been told that `const` isn't worthwhile for simple datatypes (aka "POD") passed by value like this. Here's a stackexchange thread about it:

https://stackoverflow.com/questions/1724051/const-correctness-for-value-parameters

Personally, I don't have a strong opinion either way, I think it's fine.

Reply:
This is the reason I didn't make these constant.  If you don't make them const it adds less clutter to the header and has no negative impact on the code being generated.